features/step_definitions/auction_create_steps.rb

```diff
@@ -10,6 +10,22 @@
   @auction = FactoryGirl.create(:auction, :closed, :with_bidders)
 end
 
+Given(/^I have won an auction$/) do
+  @auction = FactoryGirl.build(:auction, :closed, :with_bidders)
+  bids = @auction.bids.sort_by(&:amount)
+  b = bids.first
```

Comment:
can we use `WinningBid` here or do something similar so we are *sure* this is the winning bid?